packages/viewer/src/lib/transforms/transformation-delta.ts

```diff
@@ -0,0 +1,77 @@
+import { Matrix4, Quaternion, Vector3 } from '@vertexvis/geometry';
+
+export const ALMOST_ONE = 0.9999;
+/**
+ * There are an infinite number of orthogonal vectors, this function is used to choose one.
+ * @param normal
+ * @returns
+ */
```

Comment:
nit: Could we update this comment to explain a bit more about this method? Maybe something along the lines of 
```typescript
/**
 * For any single vector, there are an infinite number of potential orthogonal vectors. This function will determine 
 * one orthogonal vector by crossing the provided vector with a unit vector in the positive X, Y, or Z directions.
 */
```